runtime/connectors/https/https.go

```diff
@@ -74,11 +75,32 @@ func (c connector) ConsumeAsFiles(ctx context.Context, env *connectors.Env, sour
 		return nil, fmt.Errorf("failed to fetch url %s: %s", conf.Path, resp.Status)
 	}
 
-	file, err := fileutil.CopyToTempFile(resp.Body, source.Name, extension)
+	file, err := fileutil.CopyToTempFile(resp.Body, "", source.Name, extension)
 	if err != nil {
 		return nil, err
 	}
-	return []string{file}, nil
+	return &httpIterator{file: file}, nil
+}
+
+type httpIterator struct {
+	file  string
+	index int
+}
```

Comment:
1. Makes sense.
2. Can be for current implementation. But see no issue in keeping index as int as well. Int index seems more consistent with iterator as well.

Reply:
2. It made me scratch my head thinking "this will only return once". A bool would be more self-documenting until there's a time where multiple files might be returned

Reply:
I think now this is leaning towards the case where things are subjective. A arg named `index` in an `iterator` with type `bool` will probably make some people uncomfortable. 
To make it little more generic changed datatype of file to `[]string`. The entire file is a small code and I think it will probably be easy to find that the http package just returns one file.